chainercv/datasets/voc/voc_detection_dataset.py

```diff
@@ -124,7 +125,12 @@ def get_example(self, i):
             label.append(voc_utils.voc_detection_label_names.index(name))
         bbox = np.stack(bbox).astype(np.float32)
         label = np.stack(label).astype(np.int32)
-        difficult = np.array(difficult, dtype=np.bool)
+        if self.use_difficult:
```

Comment:
No, I mean that `difficult = np.array(difficult, dtype=np.bool)` is enough. If `use_difficult` is False, `difficult` becomes `[False, False, ... ]` because it skips True GTs (https://github.com/yuyu2172/chainercv/blob/52714429cc7b87c12a5f40a49f8a58c77149c66e/chainercv/datasets/voc/voc_detection_dataset.py#L116-L117). So, we can use `difficult = np.array(difficult, dtype=np.bool)` to make a False array.